src/TraceEvent/EventPipe/EventPipeEventSource.cs

```diff
@@ -844,39 +968,59 @@ protected unsafe override void ReadBlockContents(PinnedStreamReader reader)
         }
     }
 
+    internal enum EventPipeMetaDataVersion
+    {
+        LegacyV1 = 1, // Used by NetPerf version 1
+        LegacyV2 = 2, // Used by NetPerf version 2
+        NetTraceV1 = 3, // Used by NetPerf version 3 and NetTrace version 1 onwards
+    internal enum EventPipeMetaDataVersion
```

Comment:
Looks like there is a merge error here as EventPipeMetadataVersion is nested inside another definition of EventPipeMetadataVersion.

Reply:
There were a couple of noah's suggestions that caused build breaks, those should all be fixed now